configurations/code_schemes.py

```diff
@@ -41,6 +41,9 @@ class CodeSchemes(object):
     KAKUMA_HOUSEHOLD_LANGUAGE = _open_scheme("kakuma_household_language.json")
     KAKUMA_LOCATION = _open_scheme("kakuma_location.json")
 
+    KAKUMA_LEARNING_FROM_HOME_EXPERIENCE = _open_scheme('kakuma_learning_from_home_experience.json')
```

Comment:
" for consistency with the surrounding lines

Reply:
updated thanks!